packages/ui-components/LICENSE.md

```diff
@@ -0,0 +1,201 @@
+                                 Apache License
```

Comment:
I'm not 100% sure we can have this license here, i'll create a chat with us and greg